.github/workflows/Image-build-e-sign-svc.yaml

```
@@ -118,4 +126,4 @@ jobs:
           -H "Accept: application/vnd.github.v3+json" \
           -H "Authorization: token ${{ secrets.GHUB_TOKEN }}" \
           https://api.github.com/repos/pucardotorg/pucar-Devops/actions/workflows/${{ env.WORKFLOW_FILE }}/dispatches \
-          -d '{"ref":"Solutions-Pipeline"}'
+          -d '{"ref":"Solutions-Pipeline"}'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)